### ISSUE TITLE
Provide instructions on test environment or VM (Docker, etc.) for testing environment

### ISSUE THREAD

**Issue by abetusk** [stackernews/stacker.news#25]
Though it might be obvious to engineers that are better than myself, how to get a test environment working is not clear to me. It would be nice to have some minimal instructions on how to get a development setup working so that getting familiar with the environment is as easy as possible.

Better yet, it might be nice to provide a Docker container so that people can experiment with something that's pre-built.

I got as far as running the web service but, in the couple minutes I looked at it, I couldn't figure out how to hook up or create the database, so gave up. A little more documentation on how to get a dev environment working or providing a Docker container would help speed up this on-boarding.

**Comment by huumn** [comment]
I've had a few other requests to do this too. 

It is a fairly complicated dev setup to do ad hoc (like I do). Docker would probably be best.

**Comment by abetusk** [comment]
Setting up the lightning daemon (or connection thereto) is lower priority than setting up the database and connecting, etc. You obviously have more familiarity with the setup but I suspect there's the possibility of hooking up the database without the lnd connection but with some DB connection.

If true, that allows testing of the features that don't directly depend on the lightning daemon, like API access, RSS feed updates, etc.

**Comment by huumn** [comment]
You're right, you can run the app without lnd being connected. Off the top of my head, you mostly just need

1. a postgres db
2. an .env file with the `DATABASE_URL` like
```bash
# Environment variables declared in this file are automatically made available to Prisma.
# See the documentation for more detail: https://pris.ly/d/prisma-schema#using-environment-variables

# Prisma supports the native connection string format for PostgreSQL, MySQL and SQLite.
# See the documentation for all the connection string options: https://pris.ly/d/connection-strings

# prisma
DATABASE_URL="postgresql://keyan@localhost:5432/stackernews?schema=public"
``` 
3. run `prisma migrate dev`

For next auth there are other bits. I should for sure document this though.
